index.html

```diff
@@ -49,9 +49,9 @@
 						<a href="http://docs.astropy.org">Documentation</a>
 						<div class="dropdown-content">
 							<ul>
-								<li><a href="http://docs.astropy.org" target="_blank">Current Release</a></li>
+								<li><a href="http://docs.astropy.org" target="_blank">Current Release (3.1)</a></li>
 								<li><a href="http://devdocs.astropy.org/" target="_blank">In Development</a></li>
-								<li><a href="https://astropy.readthedocs.io/en/v2.0.9/index.html" target="_blank">v2.0.9</a></li>
+								<li><a href="https://astropy.readthedocs.io/en/v3.0.5/index.html" target="_blank">v3.0.5</a></li>
```

Comment:
Don't update this version to non LTS, but rather 2.0.10

Reply:
See comment below. If this is really supposed to be LTS, it should link to http://docs.astropy.org/en/lts/